### PR TITLE
[MM-32301] Post Aria Label

### PR DESCRIPTION
**Pull request by calebroseland**
#### Ticket Link

- Fixes https://mattermost.atlassian.net/browse/MM-32301

### CODE REVIEW

File: e2e/cypress/integration/messaging/message_deleted_on_reply_spec.js [hunk 1]
```diff
@@ -38,47 +40,54 @@ describe('Messaging', () => {
...
             cy.externalRequest({user: admin, method: 'DELETE', path: `posts/${postId}`});
-        });
 
-        // # Wait for the message to be deleted and hit enter
-        cy.wait(TIMEOUTS.HALF_SEC);
-        cy.get('#reply_textbox').type('{enter}');
+            // # Wait for the message to be deleted
+            cy.wait(TIMEOUTS.HALF_SEC);
+
+            // * Aria labels should contain not contain original message
```
Comment (agarciamontoro) [inline e2e/cypress/integration/messaging/message_deleted_on_reply_spec.js:49]: Typo:
```suggestion
            // * Aria labels should not contain original message
```

**Review by agarciamontoro — APPROVED** [review]
Looks good to me! Thanks, Caleb :)

**Review by devinbinnie — APPROVED** [review]
Looks good, might this be worth writing a test for?

**Comment by calebroseland** [comment]
> Looks good, might this be worth writing a test for?

@devinbinnie yes and I tried briefly, but I think needs a full mount to be able to test the lifecycle methods, and there's a bunch of connected components in this tree.

**Comment by devinbinnie** [comment]
> > Looks good, might this be worth writing a test for?
> 
> @devinbinnie yes and I tried briefly, but I think needs a full mount to be able to test the lifecycle methods, and there's a bunch of connected components in this tree.

No worries, might be worth an E2E test at some point but we can worry about that later.

**Comment by calebroseland** [comment]
> > @devinbinnie yes and I tried briefly, but I think needs a full mount to be able to test the lifecycle methods, and there's a bunch of connected components in this tree.

There's one in here :)

**Comment by devinbinnie** [comment]
> > > @devinbinnie yes and I tried briefly, but I think needs a full mount to be able to test the lifecycle methods, and there's a bunch of connected components in this tree.
> 
> There's one in here ✓

Must have missed that XD

**Review by josephbaylon — APPROVED** [review]
LGTM. Test passed locally. Thanks @calebroseland !!

**Review by ogi-m — APPROVED** [review]

**Pull request merged**